angular/src/app/app-routing.module.ts

```diff
@@ -4,14 +4,17 @@ import {UserPostContainerComponent} from "./components/user-post-components/user
 import {
   LeaguePostContainerComponent
 } from "./components/league-posts/league-post-container/league-post-container.component";
+import {LeaguePotsDetailComponent} from "./components/league-posts/league-pots-detail/league-pots-detail.component";
 import {UserPostDetailComponent} from "./components/user-post-components/user-post-detail/user-post-detail.component";
 import {ProfilePageComponent} from "./components/user/profile-page/profile-page.component";
 
 const routes: Routes = [
   {path: 'user-posts', component: UserPostContainerComponent},
   {path: 'league-posts', component: LeaguePostContainerComponent},
+  {path: 'league/:id', component: LeaguePotsDetailComponent},
```

Comment:
I think the 'all' and 'unique' paths should be the same

e.g. /user-posts, /user-posts/:id

Reply:
ohh... okay